internal/jet/select_lock.go

```diff
@@ -37,6 +44,23 @@ func (s *selectLockImpl) SKIP_LOCKED() RowLock {
 func (s *selectLockImpl) serialize(statement StatementType, out *SQLBuilder, options ...SerializeOption) {
 	out.WriteString(s.lockStrength)
 
+	if len(s.of) > 0 {
+		out.WriteString("OF")
+
+		for i, of := range s.of {
+			if i > 0 {
+				out.WriteString(",")
+			}
+
+			table := of.Alias()
+			if table == "" {
+				table = of.TableName()
+			}
+
+			out.WriteIdentifier(table, true)
```

Comment:
Why `alwaysQuote` table name?

Reply:
I think this was left over from me experimenting with what the expected behaviour was. For some reason, the tests always ended up quoting regardless of what this value was.

Will update to false as it doesn't seem always necessary.

```suggestion
			out.WriteIdentifier(table, false)
```